packages/mdc-list/mdc-list.scss

```diff
@@ -87,7 +87,9 @@
   @include mdc-list-item-primary-text-ink-color(text-disabled-on-background);
 }
 
-.mdc-list-item__graphic {
+// Extra specificity is to override .material-icons display style if used in conjunction with mdc-list-item__graphic
```

Comment:
Would it make sense to only increase specificity for `display: inline-flex`?

E.g.:

```scss
.mdc-list-item__graphic {
  @include mdc-list-graphic-size_(24px);

  flex-shrink: 0;
  align-items: center;
  justify-content: center;
  fill: currentColor;
}

// Extra specificity is to override .material-icons display style if used in conjunction with mdc-list-item__graphic
// stylelint-disable plugin/selector-bem-pattern
.mdc-list .mdc-list-item__graphic {
  display: inline-flex;
}
// stylelint-enable plugin/selector-bem-pattern
```